docs/modules/operation/pages/topology/enlinkd/introduction.adoc

```diff
@@ -22,6 +22,34 @@ The following sections describe the Enlinkd daemon and its configuration.
 Additionally, the supported link discovery implementations are described as well as a list of the SNMP MIBs that the SNMP agents must expose for EnLinkd to gather links between nodes.
 You can view detailed information about a node's connections (discovered links) and supporting link data on the node detail page within the {page-component-title} Web UI.
 
+[[ga-enlinkd-layers]]
+== Topology Layers
+
+Enhanced Linkd Topology Map supports Layers protocol maps.
+
+Click the Layers Button on the right panel of the Topology Map to access available Layers.
+
+Supported Layers Maps:
+
+*All* _(Selected by default)_ displays the topology information discovered by the Enhanced Linkd,
```

Comment:
```suggestion
* *All* (default) – displays the topology information Enhanced Linkd discovers, for all supported protocols (OSPF, ISIS, Bridge, LLDP, and CDP).
It also includes a user-defined option. 
```
These should be a bulleted list. 
Is the user-defined option to customize?

Reply:
Yup, seems reasonable.